source/interact-data/text-search.txt

```diff
@@ -0,0 +1,212 @@
+includes a text index on the ``description`` field:
+
+.. literalinclude:: /includes/interact-data/text-search.rb
+   :start-after: start-text-index-model
+   :end-before: end-text-index-model
+   :language: ruby
+   :emphasize-lines: 7
+   :dedent:
+
+.. note::
+
+   You must specify the index type as a string, as shown by ``'text'``
+   in the preceding code.
+
+Create the Text Index
+---------------------
+
+Next, you must create the text index in your collection. You can
+create the index by using an interface such as the :atlas:`Atlas UI
+</atlas-ui/indexes>` or :compass:`Compass </indexes>`. If you are
+developing your application by using Rails, you can run the following
+Rake task to create the index based on your model specification:
+
+.. code-block:: bash
+
+   bundle exec rake db:mongoid:create_indexes
+
+Perform Text Searches
+---------------------
+
+To perform a text search, use the ``$text`` evaluation query operator,
+followed by the ``$search`` field in your query filter. The ``$text`` operator
+performs a text search on the text indexed fields. The ``$search`` field
+specifies the text to search in the text indexed fields. To learn more
+about this operator, see the :manual:`$text reference
+</reference/operator/query/text>` in the {+server-manual+}.
+
+.. _mongoid-term-search:
+
+Search by a Term
+~~~~~~~~~~~~~~~~
+
+To search for a term, specify the term as a string in your query filter.
+To search for multiple terms, separate each term with spaces in the string.
+
+.. note:: Searching for Multiple Terms
+
+   When searching for multiple terms, {+odm+} returns
+   documents with at least one of the terms in text indexed fields.
+
+   Suppose you search by using the string ``'cake coffee cream'``. The
+   following list describes values that match this text query:
+
+   - ``'Has strong coffee notes.'``
+   - ``'Good for creamy coffee fans.'``
+   - ``'A rich but light cake.'``
+   - ``'A creamy coffee cake with cranberries.'``
+
+The following example runs a text search for ``description`` values that contain
+the term ``'herb'``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/interact-data/text-search.rb
+      :start-after: start-term
+      :end-before: end-term
+      :language: rust
+      :dedent:
+
+   .. output::
+      :language: none
+      :visible: false
+
+      # Sample output
+      {"_id":"...","description":"A bright, herb-based salad. A perfect starter for vegetarians and vegans.","name":"Kale Tabbouleh"}
+      {"_id":"...","description":"Grilled whole fish stuffed with herbs and pomegranate seeds. Serves 3-4.","name":"Herbed Whole Branzino"}
+      
+.. tip::
+
+   Although the search term was ``'herb'``, the method also matches
+   descriptions containing ``'herbs'`` because a MongoDB text index uses *suffix
+   stemming* to match similar words. To learn more about how
+   MongoDB matches terms, see :manual:`Index Entries
+   </core/index-text/#index-entries>`.
```

Comment:
Suggestion: Would [this](https://www.mongodb.com/docs/manual/core/indexes/index-types/index-text/text-index-) link be more descriptive? The link you use here strikes me as a little general for how you introduce it in this text (i.e. doesn't go into detail on how MongoDB matches terms)

Reply:
Did you mean this link? https://www.mongodb.com/docs/manual/core/indexes/index-types/index-text/text-index-properties/
I can change.

Reply:
Yes, sorry for cutting off the link!